documentation/modules/deploying/proc-deploying-kafkaconnector.adoc

```diff
@@ -30,16 +35,63 @@ spec:
   class: org.apache.kafka.connect.file.FileStreamSourceConnector <3>
   tasksMax: 2 <4>
   config: <5>
+    tasks.max: 1500
```

Comment:
I think this is not really used since this is covered by the `tasksMax` on line 36.

Reply:
Ah, I see.